src/CitizenAction/CitizenActionManager.php

```diff
@@ -89,4 +94,15 @@ public function populateRegistrationWithAdherentsInformations(EventRegistrationC
 
         return $eventsRegistrationHydrated;
     }
+
+    public function unregisterFromCitizenAction(CitizenAction $citizenAction, Adherent $adherent): void
+    {
+        if ($registration = $this->eventRegistrationManager->searchRegistration($citizenAction, $adherent->getEmailAddress(), $adherent)) {
```

Comment:
Tiny little detail. Inverting the logic so that the throw logic would be in the `if` section. Like this the `else` is removed and the code is more readable.